client/cloudAccount_service.go

```diff
@@ -0,0 +1,141 @@
+package client
+
+import (
+	"fmt"
+
+	"github.com/ciscoecosystem/aci-go-client/container"
+	"github.com/ciscoecosystem/aci-go-client/models"
+)
+
+func (sm *ServiceManager) CreateAccount(vendor string, account_id string, tenant string, nameAlias string, cloudAccountAttr models.AccountAttributes) (*models.Account, error) {
```

Comment:
CreateAccount -> CreateCloudAccount

Also adapt all other fct to make similar change